live_view_studio/lib/live_view_studio_web/live/light_live.ex

```diff
@@ -0,0 +1,95 @@
+defmodule LiveViewStudioWeb.LightLive do
+  @moduledoc """
+    #A LiveView module needs to define three callback functions:
+    *mount assigns the initial state of the LiveView process
+    *handle_event changes the state of the process
+    *render renders a new view for the newly-updated state
+  """
+  use LiveViewStudioWeb, :live_view
+
+  @doc """
+    #the mount callback it's the first callback that's invoked when the request comes in throught the router
+    *params is a map containing the current query params, as well as any router parameters
+    *session is a map containing private session data
+    *socket is a struct representing the websocket connection
+  """
+  def mount(_params, _session, socket) do
+    #socket strcut sotre the state of a LiveView process
+    socket = assign(socket, :brightness, 10)
+    #Calling assign this way returns a new socket struct with :brightness set to 10
+    {:ok, socket}
+  end
+
+  @doc """
+    The render function takes assigns which is the map of key/value pairs
+    we assigned to the socket in mount
+  """
+  def render(assigns) do
```

Comment:
@keliumJU what is the purpose of the render callback?

Reply:
render takes care of updating the static files every time the state changes